readme.txt

```diff
@@ -50,6 +50,10 @@ The four phases of the project are Editing, Customization, Collaboration, and Mu
 - <a href="https://wordpress.org/gutenberg/handbook/">Documentation: Creating Blocks, Reference, and Guidelines</a>
 - <a href="https://wordpress.org/gutenberg/handbook/reference/faq/">Additional frequently asked questions</a>
 
+= Where can I see Gutenberg plugin versions included in WordPress? =
```

Comment:
This is nitpicky but I like to be specific: "Where can I see which Gutenberg plugin versions are included in each WordPress release?"

Reply:
👍